downstream/modules/platform/proc-create-controller-form-view.adoc

```diff
@@ -0,0 +1,19 @@
+[id="proc_create-controller-form-view_{context}"]
+
+= Create-{ControllerName}-form-view
+
+== Using the Form View
+
+. Ensure *Form view* is selected. It should be selected by default.
+. Enter the name of the new controller.
+. Add any labels necessary.
+. Click *Advanced configuration*.
+. Enter *Hostname* of the instance. The hostname is optional. The default hostname will be generated based upon the deployment name you have selected.
```

Comment:
```suggestion
. Optional: Enter *Hostname* of the instance. The default hostname will be generated based upon the deployment name you have selected.
```
See IBM style guide in the [Procedures](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures) section, specifically In the "Indicating optional and conditional steps" section.